utils/kubernetes/describe/describe.go

```diff
@@ -57,6 +64,12 @@ const (
 	EndpointSlice
 )
 
+/*
```

Comment:
Replace to `// The ResourceMap variable contains the GroupKind information of all the Kubernetes resources that can be described.`